packages/botonic-plugin-contentful/bin/migrations/migrate_contents.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/zsh
+set -e
+BIN_DIR=${0:a:h}
+
+if [[ $# != 1 ]]; then
+  echo "Usage: $0 migration_file.ts"
+  echo "Example: $0  bin/migrations/content-types/text.ts"
+  echo
+  echo "Script to add new content types or fields"
+  echo "Be aware that some content-types migrations are WIP. Review before executing them"
+  echo "To populate a new space, use bin/import-contentful-models.sh"
+  exit 1
+fi
+
+
+SPACE_ID=$CONTENTFUL_TEST_SPACE_ID
+MANAGE_TOKEN=$CONTENTFUL_TEST_MANAGE_TOKEN
```

Comment:
because they were thought to be used only for the QA test space. you're right that it might make more sense to move to command line arguments if needed for other spaces (out of the scope of this PR)